libretro-common/include/glsm/glsmsym.h

```diff
@@ -435,7 +435,7 @@ void rglTexImage3D(	GLenum target,
  	const GLvoid * data);
 void rglTexImage2DMultisample( 	GLenum target,
   	GLsizei samples,
-  	GLenum internalformat,
+  	GLint internalformat,
```

Comment:
same as the other change

Reply:
I successfully built the mupen64 core library after incorporating your changes, but the core itself crashes Retroarch.

My toolchain is XCode 8.2, under El Capitan 10.11.6. Is my platform too old for GlideN64?

Reply:
Where does it crash?

Reply:
I captured the log, see https://pastebin.com/KNfLe6kJ for details.

Reply:
OK, got it to run, it was HAVE_THR_AL missing to get angrylion. The HLE renderers still crash. Try adding HAVE_THR_AL and force LLE=1 and angrylion in the config.

Reply:
@kivutar Thanks for that, it runs here now after forcing angrylion.
However, performance seems much worse than the parallel64 plugin.
I assume this is because I cannot run GLideN64.

Reply:
Angrylion has always been slow, accurate but slow.